CONTRIBUTING.md

```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
 
 ## Pull Requests
 
-This project uses Travis to check that it always builds on all the supported
-platforms. However, Microsemi's IT department stubbornly refuses (for silly
-reasons) to turn it on for the main repository. Therefore, in order to ensure
-your contributions get tested, please submit all pull requests to
-[lsgunth/switchtec-user][2]. From there we will ensure your contribution
-gets merged into the main repository.
+Please set the base branch to microsemi/devel when create the pull request.
+Make sure the comparing changes inclue your bug fixes or new features, the
+patches has no obvious style problems and is ready for submission when
```

Comment:
Should be "patches have no obvious..."

Also, I think you should state the actual style requirements here: we follow kernel style as documented [here](https://www.kernel.org/doc/html/latest/process/coding-style.html).

Running the checkpatch script should be a recommendation to ensure this. Especially seeing it's not included in this tree.